events/b_1_ipfs-impls.toml

```
@@ -97,9 +97,9 @@ description=""
 
 [[timeslots]]
 startTime="14:15"
-speakers=["@expede?"]
+speakers=["@expede","@matheus23"]
 title="WNFS"
-description=""
+description="The WebNative File System adds versioning, recursive encryption, and eventual consistency on top of IPFS"
```

"Recursive encryption" to me sounds too much like [re-encrypting ciphertexts](https://security.stackexchange.com/questions/13254/where-can-we-learn-more-about-recursive-encryption-schemes), which isn't that exciting. Maybe something like "cryptree-based encryption"?

```suggestion
description="The WebNative File System adds versioning, cryptree-based encryption, and eventual consistency on top of IPFS"
```